docs/node-mixin/rules/rules.libsonnet

```diff
@@ -0,0 +1,116 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'node-exporter.rules',
+        rules: [
+          {
+            // This rule gives the number of CPUs per node.
+            record: 'instance:node_num_cpu:sum',
+            expr: |||
+              count without (cpu) (
+                sum without (mode) (
+                  node_cpu_seconds_total{%(nodeExporterSelector)s}
+                )
+              )
+            ||| % $._config,
+          },
+          {
+            // CPU utilisation is % CPU is not idle.
+            record: 'instance:node_cpu_utilisation:avg_rate1m',
+            expr: |||
+              1 - avg without (cpu, mode) (
+                rate(node_cpu_seconds_total{%(nodeExporterSelector)s, mode="idle"}[1m])
+              )
+            ||| % $._config,
+          },
+          {
+            // This is CPU saturation: 1min avg run queue length / number of CPUs.
+            // Can go over 1. >1 is bad.
```

Comment:
Whether >1 is bad depends on the machine and usage, a high load is fine on a disk server for example.

Load is usually only useful when compared to its previous values, the absolute value has little meaning without that context

Reply:
I removed the `>1 is bad` as that's clearly not generally true. I have also added a TODO to vet how to use the load average here. As said above, I'd like to postpone this discussion and detach it from this PR.